### DISCUSSION TITLE
Data availability

### DISCUSSION THREAD

**Discussion by VeruGHub** [VeruGHub/easyclimate#44] [Announcements]
Climatic data are temporary unavailable because of a problem with the server. We are working on restoring them🪛⚙️
- v4 (defaut data version after easyclimate v0.2) will be available during the next days. We will keep you posted!
- v3 will take a bit longer to be available.
We apologize for the inconveniences!🙏

**Comment by cpucher** [comment]
Dear all,

finally the v4 climate data has been restored on the server and is again accessible with the easyclimate package.
Unfortunatelly v3 will take longer to restore.

Again, sorry for the inconvenience!
